frontend/modules/admin/src/lib/lib.routes.ts

```diff
@@ -39,13 +39,13 @@ export const adminRoutes: Route[] = [
       {
         title: translatedTitle('KEYWORDS'),
         path: 'keywords',
-        canMatch: [AuthorizedOfTypeGuard(ExtendedAccountTypeEnum.ADMIN)],
+        canActivate: [AuthorizedOfTypeGuard(ExtendedAccountTypeEnum.ADMIN)],
```

Comment:
non-blocking:
Why was canMatch replaced with canActivate?

Reply:
Leftover after debugging why for example `/admin` url was always redirected to `/` even if user was logged in, I'll rollback this change